lib/kitchen/driver/openstack.rb

```diff
@@ -106,10 +104,10 @@ def create(state)
       def destroy(state)
         return if state[:server_id].nil?
 
-        config[:disable_ssl_validation] && disable_ssl_validation
+        disable_ssl_validation if config[:disable_ssl_validation]
```

Comment:
So it's mentioned in the README, about being turned on ONLY if you know what you're doing. I don't think it's supposed to be used or acknowledged without specific reasons.